tagging/templatetags/tagging_tags.py

```diff
@@ -14,10 +14,10 @@ def __init__(self, model, context_var, counts):
         self.counts = counts
 
     def render(self, context):
-        model = get_model(*self.model.split('.'))
+        model = apps.get_model(*self.model.split('.'))
         if model is None:
             raise TemplateSyntaxError(_('tags_for_model tag was given an invalid model: %s') % self.model)
-        context[self.context_var] = Tag.objects.usage_for_model(model, counts=self.counts)
+        context[self.context_var] = list(Tag.objects.usage_for_model(model, counts=self.counts))
```

Comment:
whats the reason for `list` here?

Reply:
Yea, the __str__ method of a QuerySet is different in django 1.11.

Reply:
what does `str` have to do here?

Reply:
We are rendering the result of this as a string in a template. It's the new test added. I really only added it so this file would be imported (showing problems with django 1.11 imports).

Reply:
ok, that test was a big foggy too..
creating 1 instance with 1 tag didnt felt like smoke testing.